eclipse/tests/src/nl/tudelft/watchdog/util/WatchDogUtilsTest.java

```diff
@@ -64,37 +64,37 @@ public void testCountSLOC() {
 	}
 
 	@Test
-	public void testHashFileNameEmpty() {
+	public void file_name_hash() {
 		String expectedHash = "";
 		assertEquals(expectedHash, WatchDogUtils.createFileNameHash(""));
 		assertEquals(expectedHash, WatchDogUtils.createFileNameHash(null));
 	}
 
 	@Test
-	public void testFileExtensionReplace() {
+	public void hash_ignores_extension() {
 		String fileName = "ATestClass";
 		String expected = WatchDogUtils.createFileNameHash(fileName + ".java");
 		assertEquals(expected, WatchDogUtils.createFileNameHash(fileName));
 	}
 
 	@Test
-	public void testMavenTestFileNotation() {
+	public void hash_can_handle_maven_notation() {
 		String fileName = "ATestClass";
 		String expected = WatchDogUtils.createFileNameHash("org.some.package."
 				+ fileName);
 		assertEquals(expected, WatchDogUtils.createFileNameHash(fileName));
 	}
 
 	@Test
-	public void testBugReplaceWithinStringDoesNotOccur() {
+	public void replace_within_string_does_not_throws_off_hash() {
 		String unexpectedHash = WatchDogUtils
 				.createFileNameHash("AClassTest.java");
 		assertTrue(!WatchDogUtils.createFileNameHash("ATestClassTest.java")
 				.equals(unexpectedHash));
 	}
 
 	@Test
-	public void testHashFileName() {
+	public void test_should_be_extracted_into_hash() {
```

Comment:
Begins with "test" Maybe modify regex in checkstyle.xml to not be allowed to start with test `^(!test)`?

Reply:
Done :+1: